include/zephyr/net/wifi_mgmt.h

```diff
@@ -229,7 +229,9 @@ struct wifi_scan_params {
 	 *  not conforming to regulatory restrictions etc. The invoker of the API should
```

Comment:
The "Co-authored-by:" does not have a proper mail address, I think it should be
```
Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>
```

Reply:
Ah, I guess my review comment was committed and squashed from web, just removing that line is fine.

Reply:
I see, wasn't sure what that button really did when I clicked it, removed the line now tho.